src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double height;
+    private double firstLeg;
+    private double secondLeg;
+
+    public IsoscelesTrapezoid(String color, double height, double firstLeg, double secondLeg) {
+        super(color);
+        this.height = height;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        return ((firstLeg + secondLeg) / 2) * height;
+    }
+
+    public void draw() {
```

Comment:
The method `draw()` should not be responsible for calculating the area. Consider separating the concern of drawing from the concern of calculating the area.